app/decorators/case_transition_decorator.rb

```diff
@@ -19,7 +19,12 @@ def event_and_detail
 
   private
   def event
-    Case::FOI::StandardStateMachine.event_name(object.event)
+    state_machine = object.case.state_machine
+    if state_machine.is_a?(ConfigurableStateMachine::Machine)
+      state_machine.event_name(object.event)
+    else
+      Case::FOI::StandardStateMachine.event_name(object.event)
```

Comment:
Let's try to remember to add a `Case::FOI::StandardStateMachine#event_name` method that calls this class method `Case::FOI::StandardStateMachine.event_name` here. That way, we'll be able to get rid of all this code and just return `state_machine.event_name(object.event)` here.

Not required for this PR.